### PR TITLE
Display flatpage children

### PR DESCRIPTION
**Pull request by dtrucs**
- Use source dictionaries instead of fetching it (performance)
- Fix bug occurs sometimes with sources
- Define h1 tag for title page
- Display children CTA's if defined

### CODE REVIEW

File: frontend/src/components/pages/flatPage/FlatPage.tsx [hunk 1]
```diff
@@ -98,6 +99,35 @@ export const FlatPageUI: React.FC<FlatPageUIProps> = ({ flatPageUrl }) => {
...
               </>
             )}
+            {flatPage.children && flatPage.children.length > 0 && (
+              <>
+                <Separator />
+                <h2 className="my-6 desktop:my-10 text-Mobile-H1 desktop:text-H2 font-bold">
+                  <FormattedMessage id="page.children.title" />
+                </h2>
+                <ul className="mb-6 desktop:mb-18 flex flex-wrap gap-5 desktop:grid desktop:grid-cols-3 desktop:gap-6">
+                  {flatPage.children?.map(child => (
```
Comment (mabhub) [inline frontend/src/components/pages/flatPage/FlatPage.tsx:109]:
```suggestion
                  {flatPage.children.map(child => (
```

File: frontend/src/modules/flatpage/connector.ts [hunk 2]
```diff
@@ -12,18 +12,27 @@ export const getFlatPages = async (language: string): Promise<MenuItem[]> => {
 export const getFlatPageDetails = async (
   id: string,
   language: string,
+  commonDictionaries?: CommonDictionaries,
 ): Promise<FlatPageDetails> => {
+  const { sources = {} } = commonDictionaries ?? {};
```
Comment (mabhub) [inline frontend/src/modules/flatpage/connector.ts:17]:
```suggestion
  const { sources = {} } = commonDictionaries || {};
```

File: frontend/src/modules/flatpage/connector.ts [hunk 3]
```diff
@@ -12,18 +12,27 @@ export const getFlatPages = async (language: string): Promise<MenuItem[]> => {
...
     console.error('Error in flatpage connector', e);
     throw e;
   }
+  try {
+    rawFlatPageChildrenDetails = await fetchChildrenFlatPageDetails({ language }, id);
+    rawFlatPageChildrenDetails = rawFlatPageChildrenDetails.results;
+    // Old version of flatPage don't have `children` property
+  } catch (e) {
+    rawFlatPageChildrenDetails = [];
+    throw e;
```
Comment (mabhub) [inline frontend/src/modules/flatpage/connector.ts:31]: Vu ensemble : « oups » ?

**Review by mabhub — APPROVED** [review]

File: frontend/src/modules/flatpage/connector.ts (hunk 3 above)
Comment (dtrucs) [reply frontend/src/modules/flatpage/connector.ts]: Fixed!

**Review by mabhub — APPROVED** [review]

**Pull request merged**